addons/mail/static/src/js/core/dialog.js

```diff
@@ -0,0 +1,22 @@
+/** @odoo-module **/
+
+import Dialog from 'web.Dialog';
+
+Dialog.include({
+    /**
+     * @private
+     * @override
+     * @param {jQueryEvent} ev
+     */
+    _onFooterButtonKeyDown(ev) {
+        switch(ev.which) {
+            case $.ui.keyCode.TAB:
+                const name = ev.target.getAttribute('name');
+                if (['action_close_dialog', 'action_done', 'action_done_schedule_next'].includes(name)) {
+                    return;
+                }
```

Comment:
This is a bit confusing to compare names, but I guess ok as part of the old widget.